usb/usb_host_ch34x_vcp/idf_component.yml

```diff
@@ -2,8 +2,14 @@
 version: "2.0.0"
 description: USB Host driver for CH34x series of chips
 url: https://github.com/espressif/idf-extra-components/tree/master/usb/usb_host_ch34x_vcp
+
 dependencies:
   espressif/usb_host_cdc_acm:
     version: ">=1.0.4,<3.0.0"
     public: true
   idf: ">=4.4"
+
+targets:
+  - esp32s2
+  - esp32s3
+  - esp32p4
```

Comment:
The reason why I did not add the `targets` to `usb_host_xxx_vcp` components is that the targets requirement are taken from `usb_host_cdc_acm` on which they depend.

I did not want to update all the drivers for a new target, but only the base `usb_host_cdc_acm`.

Does it make sense? can we remove it?

Reply:
Yes, when I started to add them in the end this popped up in my head.... that that was the reason. 
Agreed.